scripts/install.sh

```diff
@@ -6,8 +6,22 @@ err_report() {
     echo "yvm failed to install, the command that failed was: https://github.com/tophat/yvm/blob/master/scripts/install.sh#L${1}"
 }
 
+err_report_missing_dependency() {
+    echo "The install cannot proceed due to a missing dependency. \"${1}\" is not installed."
+}
+
 trap 'err_report $LINENO' ERR
 
+INSTALL_DEPENDENCIES=( "unzip" "curl" )
+
+for package in "${INSTALL_DEPENDENCIES[@]}"
+do
+    if [ ! `eval "$package -v"` ]; then
```

Comment:
Have yet to QA, but `curl -v` does not exit with `0`

Reply:
That's a good point. `--version` however is common to both the packages specified, so it should return with a 0 if it's there.

```suggestion
    if [ ! `eval "$package --version"` ]; then
```

Reply:
I take it back, `--version` doesn't play well with `unzip`. I'll look into it tomorrow to find a better check.

Reply:
In a subsequent commit, I updated this to use `command -v <command>`, which should either give you the path to the executable or nothing, which works well with the `if` we have here.